frontend/src/views/AccessibilityDeclaration.vue

```diff
@@ -223,7 +223,12 @@
     </h3>
     <p>
       Vous pouvez suivre notre progrès sur l'amélioration de nos non-confirmités sur
-      <a href="https://github.com/betagouv/ma-cantine/issues?q=is%3Aissue+is%3Aopen+label%3Aa11y">
+      <a
+        href="https://github.com/betagouv/ma-cantine/issues?q=is%3Aissue+is%3Aopen+label%3Aa11y"
+        target="_blank"
+        rel="noopener external"
+        title="GitHub issus - ouvre une nouvelle fenêtre"
```

Comment:
`issu` veut plutôt dire venant de quelque part
```suggestion
        title="GitHub issues - ouvre une nouvelle fenêtre"
```